From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <nornagon@nornagon.net>
Date: Thu, 20 Sep 2018 17:48:38 -0700
Subject: gtk_visibility.patch

Allow electron to depend on GTK in the GN build.

diff --git a/build/config/linux/gtk/BUILD.gn b/build/config/linux/gtk/BUILD.gn
index 5491aef735fc277521acbff97c1484fa81ef9ba4..8ff2728d1843e6d436aa2c59a6b991f2f257e864 100644
--- a/build/config/linux/gtk/BUILD.gn
+++ b/build/config/linux/gtk/BUILD.gn
@@ -29,6 +29,8 @@ group("gtk") {
     # This should probably be moved into //ui/gtk.
     "//ui/ozone/platform/wayland",
 
+    "//electron:*",
+
     # These are all for WebRTC.
     "//examples:peerconnection_client",
     "//remoting/host:common",
